web/chromecom.js

```diff
@@ -275,7 +275,7 @@ var ChromeCom = (function ChromeComClosure() {
     function onDisconnect() {
       // When the connection fails, ignore the error and call the callback.
       port = null;
-      onCompleted();
+      callback();
```

Comment:
It doesn't matter. The port will be the garbage-collected, together with
the listeners.